fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_searchType.vail

```diff
@@ -0,0 +1,18 @@
+package com.vantiq.fhir
+
```

Comment:
This is likely a precursor to a real search capability.  I just added it because I needed it for the tests right now.  It's not wrong, just probably incomplete for the "real" service.